docs/nidcpower/rep_caps.rst

```diff
@@ -18,10 +18,15 @@ Repeated Capabilities
 
```

Comment:
>    Repeated capabilities attributes are used to set the `channel_string` parameter to the
    underlying driver function call. This can be the actual function based on the :py:class:`Session`
    method being called, or it can be the appropriate Get/Set Attribute function, such as :c:`niDCPower_SetAttributeViInt32()`.

The above is gibberish to a Python developer not familiar with our C API. Knowing our C API is not a requirement for using our Python APIs.

This is tricky word-smithing, so here goes something based on https://www.ivifoundation.org/resources/IVI%20Repeated%20Capabilities.pdf

> `nidcpower.Session` supports "Repeated Capabilities", which are multiple instances of the same type of
> functionality. The repeated capabilities supported by `nidcpower.Session` are:
> 1. `channels`
> 2. `instruments`
>
> Use the indexing operator `[]` to indicate which repeated capability you are trying to access. The parameter can 
> be an integer, a string, a list, a tuple, or slice (range).

Reply:
Done. With the list items linking to sections of the page!
Let me know if you don't like it.